.github/workflows/build-custom.yml

```diff
@@ -187,28 +98,28 @@ jobs:
         if ${{ github.event.inputs.runTutorialHeatedPlate }}
         then
           cd tutorials/flow-over-heated-plate/fluid-openfoam
-          ${{steps.installOpenFOAM.outputs.openfoam_exec}} ./run.sh | tee fluid-openfoam.log 2>&1 &
+          ./run.sh | tee fluid-openfoam.log 2>&1 &
```

Comment:
Have you tested if this workflow indeed works? I remember it was tricky to expose the environment (but I don't remember why anymore).

Reply:
Keeping the environment for future steps is not straightforward, but the action manages to do it by tracking the OpenFOAM variables and exposing them via `$GITHUB_ENV` and `$GITHUB_PATH`.

I've tested the workflow with my fork now and it seems to work okay: https://github.com/gerlero/openfoam-adapter/actions/runs/10489014497/job/29052697825

Reply:
Good point about the `$GITHUB_ENV` and `$GITHUB_PATH`, thanks!
Apparently, it works. We can discover any corner cases in develop.

Reply:
Also works in the main repo:

- OpenFOAM v2106 on Ubuntu 20.04, using the develop branch: https://github.com/precice/openfoam-adapter/actions/runs/10507653425
- OpenFOAM 7 on Ubuntu 20.04, using the OpenFOAM7 branch: https://github.com/precice/openfoam-adapter/actions/runs/10507736208 (the adapter building fails as currently known)